corehq/apps/userreports/reports/filters/values.py

```diff
@@ -128,35 +133,129 @@ def to_sql_values(self):
         }
 
 
+def get_dyn_range_filter(slug):
+
+    class DynRangeFilter(BasicFilter):
+        operator = slug
```

Comment:
This is not how we use `operator` in other filters. Open to suggestions on how best to keep with existing convention.